app/models/user.rb

```diff
@@ -50,7 +50,7 @@ class User < ActiveRecord::Base
   validates :state, inclusion: { in: VALID_STATES,
                                  message: "must be one of #{VALID_STATES.join(',')}" }
 
-  validate :name_part_required_for_suffix_or_title
+  validates :first_name, :last_name, presence: true, allow_blank: false
```

Comment:
We're going to have a problem with existing users who don't have first or last names.  E.g. when the Salesforce routine updates their SF information (which is cached in the User record), it won't be able to save because of this validation.  

Could do something like make the validation smarter:

```ruby
validates :names_present

def names_present
  # freak out if (first_name_was.present? && first_name.blank?) -- ditto last_name
  # different conditions for new_record?
  # something like this anyway :-)
end